spec/dish_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'dish'
+
+describe Dish do
+  subject { Dish.new("Test Dish", 10) }
+
+  it { is_expected.to respond_to(:name) }
+  it { is_expected.to respond_to(:price) }
+end
```

Comment:
Does this test if there is a value assigned to :name and :price like line 4 is indicating or that the subject is aware of the attribute?